MVVMRxSwiftDemo/Sources/Presentation/Views/AnswerView/AnswerViewProtocol.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  AnswerViewProtocol.swift
+//  MVVMRxSwiftDemo
+//
+//  Created by Chananchida F. on 7/27/21.
+//
+
+import UIKit
+import RxSwift
+
+typealias AnswerView = UIView & AnswerViewProtocol
+
+protocol AnswerField {
+    
+    var answerView: AnswerViewProtocol { get }
+}
+
+protocol AnswerViewProtocol: UIView {
+    
+    func configure(with answers: [SurveyAnswer])
+}
```

Comment:
As we have typealias `AnswerView`, I would prefer this:
```suggestion
protocol AnswerField {
    
    var answerView: AnswerView { get }
}

protocol AnswerViewProtocol {
    
    func configure(with answers: [SurveyAnswer])
}
```